benches/classic_crypto.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright © 2023 Niklas Siemer
+//
+// This file is part of qFALL-math.
+//
+// qFALL-math is free software: you can redistribute it and/or modify it under
+// the terms of the Mozilla Public License Version 2.0 as published by the
+// Mozilla Foundation. See <https://mozilla.org/en-US/MPL/2.0/>.
+
+//! Contains benchmarks for some classic cryptographic schemes.
+
+use criterion::*;
+use qfall_math::integer::Z;
+
+criterion_group!(benches, bench_rsa);
```

Comment:
Please move this line to the bottom of the file.
Criterion also has it at the bottom of their benchmarks. (https://github.com/bheisler/criterion.rs/blob/master/benches/benchmarks/iter_with_setup.rs)